.gitignore

```diff
@@ -1,7 +1,7 @@
 *.swp
 .DS_Store
 .vscode/
-build-*/
+build*/
```

Comment:
maybe add a distinct build/ entry?
build*/ could include other directories like builder/ buidifier/ buildable-pipeline or whatever random new directory people add. the actual hyphen limits the damage a bit

Reply:
added back build/